public/docs/_examples/dependency-injection/dart/lib/logger_service.dart

```diff
@@ -3,10 +3,12 @@ import 'package:angular2/core.dart';
 
 @Injectable()
 class Logger {
-  List<String> logs = [];
+  List<String> _logs = [];
+
```

Comment:
The blank line above the getter seems excessive. Is it standard? (It could be; I don't happen to know, and I don't see anything definitive in Effective Dart.)